src/JoomlaBrowser.php

```diff
@@ -899,4 +905,18 @@ public function disableStatistics()
         $I->waitForElement(['link' => 'Never'], 60);
         $I->click(['link' => 'Never']);
     }
+
+	/**
+	 * Method is to set Wait for page title
+	 *
+	 * @param   string   $title    Page Title text
+	 * @param   integer  $waiting  Waiting time
+	 *
+	 * @return  void
+	 */
+	public function waitForPageTitle($title, $timeout = 60)
+	{
+		$I = $this;
+		$I->waitForText($title, $timeout, ['class' => '.page-title']);
```

Comment:
you are right: 

['class' => 'page-title']);

dot would be only if you do:

['css' => '.page-title']);